packages/ui/src/i18n/translations.ts

```diff
@@ -85,7 +85,9 @@ export const translations: Record<string, Dict> = {
   id: idDict,
   it: itDict,
   ja: jaDict,
+  // TODO: remove kr in next major release
```

Comment:
Can you please add a ticket in the Major Release project for this?

Reply:
Done